Composer/packages/client/src/components/CreationFlow/v2/DefineConversation.tsx

```diff
@@ -70,8 +72,8 @@ type DefineConversationFormData = {
   name: string;
   description: string;
   schemaUrl: string;
-  primaryLanguage: string;
-  runtimeChoice: string;
+  runtimeLanguage: string;
+  runtimeChoice: RuntimeType;
```

Comment:
nit: runtimeType seems like an obvious choice for the prop name

Reply:
good suggestion i changed this everywhere.